modules/va_forms/config/routes.rb

```diff
@@ -4,6 +4,8 @@
   match '/metadata', to: 'metadata#index', via: [:get]
   match '/v0/*path', to: 'application#cors_preflight', via: [:options]
   match '/v0/healthcheck', to: 'metadata#healthcheck', via: [:get]
+  match '/v0/upstream_healthcheck', to: 'metadata#upstream_healthcheck', via: [:get]
+  match '/v1/upstream_healthcheck', to: 'metadata#upstream_healthcheck', via: [:get]
```

Comment:
Looks like there isn't a v1 for va_forms yet.

Reply:
Thank you, I was wondering about that.